### PR TITLE
updates to User agent

### PR DESCRIPTION
**Pull request by mikehancock**

### CODE REVIEW

File: JudoPayDotNet/Http/HttpClientWrapper.cs [hunk 1]
```diff
@@ -14,6 +15,8 @@ namespace JudoPayDotNet.Http
     // ReSharper disable UnusedMember.Global
     internal class HttpClientWrapper : IHttpClient
     {
+        internal static readonly string SdkUserAgent = "DotNetSDK/" + new AssemblyName(typeof(HttpClientWrapper).GetTypeInfo().Assembly.FullName).Version;
```
Comment (akakjs) [inline JudoPayDotNet/Http/HttpClientWrapper.cs]: How about adding the .net runtime version? and Framework/Core/Mono?

File: JudoPayDotNetTests/Headers/UserAgentTests.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    using JudoPayDotNet.Http;
+
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class UserAgentTests
+    {
+        [TestCase("Mozilla / 4.0", 1)]
+        [TestCase("Mozilla / 4.0(compatible)", 0)] // The underlying .Net classes are unable to parse these values
+        [TestCase("Mozilla / 4.0(compatible; MSIE 6.0; Windows NT 5.2; .NET CLR 1.0.3705;)", 0)] // The underlying .Net classes are unable to parse these values
```
Comment (akakjs) [inline JudoPayDotNetTests/Headers/UserAgentTests.cs]: Shouldn't we have a test case for our SDK format as well?

Comment (mikehancock) [inline JudoPayDotNetTests/Headers/UserAgentTests.cs]: will do. I was trying to get a better understanding of what passes fails the user agent parser as the documentation is a little sparse. The outcome is less than optimal and from what I can see any string with any braces, semi-colons, commas etc will fail to parse. This means that we won't get any user-agent headers set by the clients most of the time. Happy to discuss this as I can't see any easy answer, and I'm not keen to write a user-agent parser.

File: JudoPayDotNet/Http/HttpClientWrapper.cs (hunk 1 above)
Comment (mikehancock) [inline JudoPayDotNet/Http/HttpClientWrapper.cs]: good point - not sure if it's available in PCL though, certainly the machine O/S isn't

**Pull request merged**